tools/tertiary-analysis/scanpy/scanpy-parameter-iterator.xml

```diff
@@ -21,6 +27,7 @@
       <param type="float" value="0" name="start_value" label="Starting value" help="Initial value for the parameter"/>
       <param type="float" value="1" name="step" label="Step" help="Increment to use from first to last value"/>
       <param type="float" value="10" name="end_value" label="Ending value" help="Final value for the parameter. The final value is not included if increments of step don't match the last value."/>
+      <param type="text" value="0.1, 0.3, 0.5, 0.7, 1.0, 2.0, 3.0, 4.0, 5.0" name="input_values" label="User input values" help="Defined input values for the parameter"/>
```

Comment:
On the help could we be a bit more explicit, like "List all the parameter values (by comprehension) that should be iterated over.". I would additionally add something saying that this overrides the other setup (or directly make the UI expose one or the other, like it is done here: https://github.com/ebi-gene-expression-group/container-galaxy-sc-tertiary/blob/develop/tools/tertiary-analysis/ucsc-cell-browser/cell-browser.xml#L36.